vertical-pod-autoscaler/hack/vpa-process-yamls.sh

```diff
@@ -39,8 +39,17 @@ if [ $# -gt 2 ]; then
   exit 1
 fi
 
+SEMVER_RE='v[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)'
+K8S_VERSION=$(kubectl version --short=true|grep Server|awk '{print $3}')
+K8S_MINOR_VERSION=$(echo $K8S_VERSION | sed -e "s#$SEMVER_RE#\2#")
+
 ACTION=$1
-COMPONENTS="vpa-v1-crd vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
+COMPONENTS="vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
+if [ $K8S_MINOR_VERSION -lt 16 ]; then
+    COMPONENTS="vpa-v1-crd ${COMPONENTS}"
+else
+    COMPONENTS="vpa-v1-crd-v1-gen ${COMPONENTS}"
+fi
 case ${ACTION} in
 delete|diff|print) COMPONENTS+=" vpa-beta2-crd" ;;
```

Comment:
What I meant is that we need to add vpa-v1-crd here to delete it on vpa-down.sh

Reply:
Remove version-specific processing and specify only vpa-v1-crd-v1-gen; although the versions of the CRD are different, they define the same VPA resources, we cand delete CR using either crd-v1-crd or crd-v1-crd-gen files.

Reply:
Makes sence, thank you